CruCentralCoast/Login/ForgotPasswordVC.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ForgotPasswordVC.swift
+//  CruCentralCoast
+//
+//  Created by Michael Cantrell on 9/12/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+import FirebaseAuth
+
+
+class ForgotPasswordVC: UIViewController {
+    @IBOutlet weak var emailTextField: CruTextField!
+    var email: String? { didSet { self.emailTextField?.setText(self.email) } }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.emailTextField.setText(self.email)
+    }
+    
+    @IBAction func sendEmailButtonPressed(_ sender: Any) {
+        guard let email = self.emailTextField.text, !email.isEmpty else {
+            self.presentAlert(title: "Email Field Empty", message: "Type the email associated with the account that you want to reset the password of")
```

Comment:
Instead of presenting an alert, you could call the validateHasText() method on the CruTextField. This would emulate the behavior on the other two login screens.